### PR TITLE
[xdoctest][task 165] reformat example code with google style in  python/paddle/vision/ops.py

### PR DESCRIPTION
**Pull request by liyongchao911**
### PR types

Others

### PR changes

Others

### Description

修改如下文件的示例代码，使其通过 `xdoctest` 检查：

-  python/paddle/vision/ops.py

### Related links

-  #55629
-  #55295

@sunzhongkai588 @SigureMo @megemini

### CODE REVIEW

File: python/paddle/vision/ops.py [hunk 1]
```diff
@@ -1888,7 +1879,7 @@ def nms(
     Args:
         boxes(Tensor): The input boxes data to be computed, it's a 2D-Tensor with
             the shape of [num_boxes, 4]. The data type is float32 or float64.
-            Given as [[x1, y1, x2, y2], …],  (x1, y1) is the top left coordinates,
+            Given as [[x1, y1, x2, y2], ��],  (x1, y1) is the top left coordinates,
```
Comment (megemini) [inline python/paddle/vision/ops.py]: 乱码？

File: python/paddle/vision/ops.py [hunk 2]
```diff
@@ -2100,16 +2090,24 @@ def generate_proposals(
...
+            ...                 img_size, anchors, variances, return_rois_num=True)
+            >>> print(rois, roi_probs, roi_nums)
+            Tensor(shape=[2, 4], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[0., 0., 0., 0.],
+             [0., 0., 0., 0.]])
+            Tensor(shape=[2, 1], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[0.],
+             [0.]])
+            Tensor(shape=[2], dtype=int32, place=Place(cpu), stop_gradient=True,
+            [1, 1])
```
Comment (megemini) [inline python/paddle/vision/ops.py:2109]: 加个 seed 吧（好像也不管用 ... ...）～ 不然有一定概率会不是 0 ～

File: python/paddle/vision/ops.py [hunk 3]
```diff
@@ -170,25 +170,22 @@ def yolo_loss(
...
+        >>> loss = paddle.vision.ops.yolo_loss(x,
+        ...                                    gt_box=gt_box,
+        ...                                    gt_label=gt_label,
+        ...                                    anchors=[10, 13, 16, 30],
+        ...                                    anchor_mask=[0, 1],
+        ...                                    class_num=2,
+        ...                                    ignore_thresh=0.7,
+        ...                                    downsample_ratio=8,
+        ...                                    use_label_smooth=True,
+        ...                                    scale_x_y=1.)
```
Comment (SigureMo) [inline python/paddle/vision/ops.py]: 需要加缩进

File: python/paddle/vision/ops.py [hunk 4]
```diff
@@ -356,19 +353,19 @@ def yolo_box(
 
     .. code-block:: python
```
Comment (SigureMo) [inline python/paddle/vision/ops.py]: 需要加缩进

File: python/paddle/vision/ops.py [hunk 5]
```diff
@@ -2100,16 +2087,31 @@ def generate_proposals(
...
+            >>> # doctest: +SKIP('random sample')
+            >>> print(roi_probs, roi_nums)
+            Tensor(shape=[2, 1], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[0.],
+             [0.]])
+
+            >>> # doctest: -SKIP
+            >>> print(roi_nums)
+            Tensor(shape=[2], dtype=int32, place=Place(cpu), stop_gradient=True,
+            [1, 1])
```
Comment (megemini) [inline python/paddle/vision/ops.py]: 这里整体跳过就行 ～

另外，每次 print 一个就行 ～ 这每次少 print 一个 ... ...  忒抽象了 ～ 🤣🤣🤣

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾
